store/tikv/mock-tikv/cop_handler_dag.go

```diff
@@ -381,14 +390,16 @@ func reverseKVRanges(kvRanges []kv.KeyRange) {
 
 const rowsPerChunk = 64
 
-func appendRow(chunks []tipb.Chunk, handle int64, data []byte) []tipb.Chunk {
-	if len(chunks) == 0 || len(chunks[len(chunks)-1].RowsMeta) >= rowsPerChunk {
+func appendRow(chunks []tipb.Chunk, data []byte, rowCnt int) ([]tipb.Chunk, int) {
```

Comment:
I think is cleaner to manage `rowCnt` outside the function.
The function only need to know if it need to create a new chunk.
On the outside, we can pass the arguement as `rowCnt % rowsPerChunk == 0`